airbyte-integrations/connectors/source-okta/pyproject.toml

```
@@ -16,7 +16,7 @@ packages = [ { include = "source_okta" }, {include = "main.py"} ]
 
 [tool.poetry.dependencies]
 python = "^3.9,<3.12"
-airbyte-cdk = "^0"
+airbyte-cdk = "^3"
```

Did you test this and verified that anything above 4.0 breaks, or is this because you considered CDK 3.* current?

There are a lot of performance gains in CDK 5, I'd suggest attempting 5.10.2+.

```suggestion
airbyte-cdk = "^5.11"
```

---

If you do choose to accept this, you would need to re-run `poetry lock` locally and push that file.